.github/scripts/auto-label.cjs

```diff
@@ -0,0 +1,169 @@
+'use strict';
+const path = require('path');
+
+const babelParser = require('recast/parsers/babel');
+const json5 = require('json5');
+const lodash = require('lodash');
+const recast = require('recast');
+const github = require('@actions/github');
+const { HttpClient } = require('@actions/http-client');
+
+const prefixLabelMap = {
+  'resources/': 'resources',
+  'util/': 'util',
+  'test/': 'test',
+  '.mocharc.cjs': 'test',
+  'eslint/': 'style',
+  '.eslintrc.js': 'style',
+  '.github/workflows/': 'ci',
+  '.github/scripts/': 'ci',
+  'ui/config/': 'config',
+  'ui/dps/': 'dps',
+  'ui/eureka/': 'eureka',
+  'ui/fisher/': 'fishing🎣',
+  'ui/jobs/': 'jobs',
+  'ui/oopsyraidsy/': 'oopsy',
+  'ui/pullcounter/': 'pull',
```

Comment:
```suggestion
  'ui/pullcounter/': 'pullcounter',
```
@quisquous thoughts?
Also, I think we should have a `raidemulator` distinction within ui/raidboss/emulator if possible?

Reply:
`pullcounter` is fine.  Special-casing `raidemulator` sounds useful too.  Do you think a raid emulator change should be both `raidboss` and `raidemulator`?

Reply:
maybe `raidemulator` only

Reply:
It definitely could, but I think that all people interested in raidemulator changes are interested in raidboss changes, but the opposite is certainly not true, so maybe not for the purpose of this?
(Another piece might be that all people interested in raidemulator title their PRs as such, but that feels lazy to rely on people doing that)

Reply:
To summarize my thoughts above since I didn't:
I think raidemulator only for raidemulator changes is fine, or both labels is also fine.